overrides/scripts/Electronics.zs

```diff
@@ -477,7 +477,7 @@ recipes.addShaped(<gregtech:meta_item_1:12152>,[[<gregtech:meta_tool:6>],[<gregt
 
 //Resonant Clathrate
 furnace.remove(<minecraft:ender_pearl>);
-furnace.addRecipe(<forestry:crafting_material>, <thermalfoundation:material:895>, 0.0);
+furnace.addRecipe(<contenttweaker:pulsatingdust>, <thermalfoundation:material:895>, 0.0);

```

Comment:
Can furnace recipes take oredict inputs? I don't remember off the top of my head.

Reply:
This is actually the output, since furnace recipes list output first